faqs.rst

```diff
@@ -20,3 +20,82 @@ How do I report a bug?
 ----------------------
 
 You can read up on reporting bugs here: :ref:`contributing` or report one directly for flux `core <https://github.com/flux-framework/flux-core/issues>`_ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
+
+.. _not_managing_all_resources:
+
+---------------------------------------------------------------------------------
+Why is Flux not discovering and managing all of the resources on the system/node?
+---------------------------------------------------------------------------------
+
+* Various bind flags passes to the parallel launcher that started Flux
+* CUDA enabled hwloc (for GPUs)
```

Comment:
should include the changes we discussed on the phone:

- give examples for flags used at LLNL for jsrun and srun
- copy the text from the CORAL systems about cuda-enabled hwloc